web/styles/modal.css

```diff
@@ -62,6 +62,8 @@
     max-width: 80%;
     display: flex;
     align-items: center;
+    gap: 10px;
+    line-height: 28px;
```

Comment:
Let's note in the code that this line-height is to increase the vertical clickable areas on the icons. I don't love magical numbers like this, but I guess it's one-off enough to be okay.